lte/gateway/c/session_manager/SessionState.cpp

```diff
@@ -822,15 +876,64 @@ SessionConfig SessionState::get_config() const {
   return config_;
 }
 
-uint32_t SessionState::get_local_teid() const {
-  return local_teid_;
-}
-
-void SessionState::set_local_teid(
-    uint32_t teid, SessionStateUpdateCriteria& uc) {
-  local_teid_              = teid;
-  uc.is_local_teid_updated = true;
-  uc.local_teid_updated    = teid;
+uint32_t SessionState::get_upf_local_teid() const {
+  char teidar[10];
+  union mix_t {
+    std::uint32_t theDWord;
+    std::uint8_t theBytes[4];
+  };
+  mix_t aVal;
+  // Retreive value
+  std::string teids = config_.rat_specific_context.m5gsm_session_context()
+                          .upf_endpoint()
+                          .teid();
```

Comment:
defining teid as string does not make sense. Please make it uint32_t everywhere...

Reply:
same as above

Reply:
raised PR

Reply:
Added the same, Raised the Issue #5679 -->#5679